doc/usage/extensions/math.rst

```diff
@@ -264,12 +264,13 @@ Sphinx but is set to automatically include it from a third-party site.
       or "defer" key is set.
 
 .. confval:: mathjax3_config
-   :type: :code-py:`dict[str, Any] | None`
+   :type: :code-py:`str | dict[str, Any] | None`
    :default: :code-py:`None`
 
    The configuration options for MathJax v3 (which is used by default).
-   The given dictionary is assigned to the JavaScript variable
-   ``window.MathJax``.
+   Expects a string with the relative path to a javascript file with the config.
```

Comment:
```suggestion
   Expects a string with the relative path to a JavaScript file with the config.
```

Do we need to say it's relative to the config directory? I don't recall what we say for other similar cases